www/docs/en/6.x/guide/platforms/android/index.md

```diff
@@ -310,7 +310,7 @@ To sign an app, you need the following parameters:
 These parameters can be specified using the command line arguments above to
 the [Cordova CLI][cli_reference] `build` or `run` commands.
 
-__Note__: You should use double `--` to indicate that these are platform-specific arguments, for example:
+__Note__: You should use double `--` (followed by a space) to indicate that these are platform-specific arguments, for example:
```

Comment:
maybe add an "i.e. `-- --`" in the brackets as well.

Reply:
@heymartinadams Seen my comment?

Reply:
Oh gosh, I’m such a novice when it comes to GitHub pull requests. How do I do these changes now, @janpio?

Reply:
I may have deleted my fork?

Reply:
@janpio See if the other two PRs #763 and #764 do the trick.

Reply:
They do, but content is strange now.